main/src/Main.js

```diff
@@ -21,7 +21,7 @@ class Main {
      * 8. Find song X in cluster
      * 9. Get songs in cluster containing song X
      * 10. Get desired number of songs (currently 30) if cluster has too much or too little
-     * 11. Return this.result (see this.setResults)
+     * 11. return result from getResults
```

Comment:
Thanks for fixing this!! :) Feel free to add whatever to kMeans or the other files if you have better documentation

Reply:
Ah shoot it should have been capital R for return